packages/design-system/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "dataverse-design-system",
```

Comment:
The pull request description says, "This new library has been called dataverse-ui-lib."

It should be updated, right?